app/api/notifications.py

```
@@ -0,0 +1,517 @@
+                self.disconnect(user_id)
+
+
+manager = ConnectionManager()
+
+
+# Notification CRUD endpoints
+
+@router.post("/", response_model=NotificationResponse, status_code=status.HTTP_201_CREATED)
+async def create_notification(
+    notification_data: NotificationCreate,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.13.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)